packages/react/src/styles/layout.scss

```diff
@@ -0,0 +1,90 @@
+//
+//  MEDIA QUERIES
+//––––––––––––––––––––––––––––––––––––––––––––––––––
+
+// A map of breakpoints.
+$breakpoints: (
```

Comment:
It seems that breakpoints are already defined at least in 2 places already in hds, so maybe we could apply them from there instead of re-defining them here? Currently they are defined in these files:

https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/design-tokens/tokens/breakpoint/breakpoint.json (these are generated into all.scss and breakpoint.scss files)

https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/src/styles/breakpoints.css

Reply:
Could we take the variables from design-tokens and just apply them inside this file?

Reply:
Let's try! I have to improve my sass knowledge, it seems that json can be imported to the file.